homeassistant/components/zwave_js/icons.json

```diff
@@ -59,6 +59,8 @@
   "services": {
     "bulk_set_partial_config_parameters": "mdi:cogs",
     "clear_lock_usercode": "mdi:eraser",
+    "get_lock_usercode": "mdi:lock-percent-outline",
+    "get_lock_usercodes": "mdi:lock-percent",
```

Comment:
I struggled to find an appropriate icon. If you know of something more intuitive or uniform with other icons in the app, let me know!